### PR TITLE
Join table

### PR DESCRIPTION
**Pull request by Yemisirach**
- [x] Create a table named vets
- [X] Create a "join table" called specializations
- [x] Create a "join table" called visits
- [x] Insert data for vets, specialties and visits
- [x] Write queries to answer the following:
 [ 1 ] Who was the last animal seen by William Tatcher?
 [ 2 ]  How many different animals did Stephanie Mendez see?
 [ 3 ] List all vets and their specialties, including vets with no specialties.
 [ 4 ] List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
 [ 5 ]  What animal has the most visits to vets?
 [ 6 ] Who was Maisy Smith's first visit?
 [ 7 ] Details for most recent visit: animal information, vet information, and date of visit.
 [ 8 ] How many visits were with a vet that did not specialize in that animal's species?
 [ 9 ] What specialty should Maisy Smith consider getting? Look for the species she gets the
 most.
s1
s3
s4
s5

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -28,3 +28,20 @@ ALTER TABLE animals
     ADD FOREIGN KEY (species_id) REFERENCES species (id),
     ADD FOREIGN KEY (owner_id) REFERENCES owners (id);
   
+CREATE TABLE vets (
+    id                 INT  GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    name               CHAR(30),
```
Comment (danmainah) [inline schema.sql]: - [ ] @Yemisirach kindly change the column type from `char` to `varchar` to better represent the sting type of the name field.

File: data.sql [hunk 2]
```diff
@@ -45,3 +45,35 @@ INSERT INTO animals (name, species_id, owner_id) VALUES
   ('Blossom', 1, 4),
   ('Angemon', 2, 5),
   ('Boarmon', 1, 5);
+
+
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('William Tatcher', 45, '2000-04-23');
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('Maisy Smith', 26, '2019-01-17');
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('Stephanie Mendez', 64, '1981-05-04');
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('Jack Harkness', 45, '2008-06-08');
```
Comment (danmainah) [inline data.sql]: - [ ] @Yemisirach kindly check on the age of 'Jack Harkness', you have entered the wrong figure.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Yemisirach , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️

- [ ] @Yemisirach great work on your documentation however the issue link is broken kindly fix it.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED 👍 

Hi @Yemisirach , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  SQL Commands put in the correct files
:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file and PR with required screenshots

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**